ccd-definition/CaseEventToFields/createOrder-PREPARE_FOR_HEARING.json

```diff
@@ -131,7 +147,7 @@
     "CaseEventID": "createOrder-PREPARE_FOR_HEARING",
     "CaseFieldID": "orderFurtherDirections",
     "PageFieldDisplayOrder": 1,
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "MANDATORY",
```

Comment:
just double checking that we want to make this change? this field is mandatory for submitted state so I presume it should be submitted for this state too (user has to select yes/no, right?)

Reply:
Yeah, must have missed it during the copy paste when this was first implemented, I made sure that  the files for all 3 events where unified in this PR